example/.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  bracketSpacing: false,
+  jsxBracketSameLine: true,
+  singleQuote: true,
+  trailingComma: 'all',
+};
```

Comment:
Yes, I used React Native CLI. I re-created the example project from scratch and imported the necessary files since upgrade rn  projects from <=0.59.10 to 0.6x.x is very cumbersome. 

I guess, I wasn't paying enough attention to all files and the file you marked, slipped through. I'm sorry.